### PR TITLE
Week 1 Step 1 Refactored

### PR DESCRIPTION
**Pull request by lazyplatypus**

### CODE REVIEW

File: .bit/responses/1.1-Week 1 Step 1.md [hunk 1]
```diff
@@ -6,42 +6,46 @@ week: 1
...
+<summary>Vocabulary</summary>
+
+#### Repositories
 Repositories (or repos) are essentially **folders where you can store files of code.** The repo of our camp was duplicated into your account when you clicked "Create Template" so that you can commit changes and complete each lesson.
 
+#### Issues
 For our camp, each week is placed inside an issue. Only when you complete the week (committing the necessary code and commenting), will the issue close and you can move on to the next issue. Don’t worry – committing changes is easier than it sounds.
-<br><br/>
+
+#### Pull Request
```
Comment (lazyplatypus) [inline .bit/responses/1.1-Week 1 Step 1.md:41]: add information about what is a pull request

File: .bit/responses/1.1-Week 1 Step 1.md [hunk 2]
```diff
@@ -6,42 +6,46 @@ week: 1
...
+
+#### Branch
```
Comment (lazyplatypus) [inline .bit/responses/1.1-Week 1 Step 1.md:43]: add information about what is a branch!

File: .bit/responses/1.1-Week 1 Step 1.md [hunk 3]
```diff
@@ -6,42 +6,46 @@ week: 1
...
+<br>
 </details>
 
 <details>
+
 <summary>:exclamation: How can I use GitHub Desktop to commit my code?</summary>
```
Comment (lazyplatypus) [inline .bit/responses/1.1-Week 1 Step 1.md:49]: Replace this with how do I use command line to commit code

File: .bit/responses/1.1-Week 1 Step 1.md [hunk 4]
```diff
@@ -60,48 +64,24 @@ For our camp, each week is placed inside an issue. Only when you complete the we
 <br><br/>
 </details>
 
-</details>
 <details>
 <summary>:exclamation: Help! I don't know how to use it and I need more information.</summary>
```
Comment (lazyplatypus) [inline .bit/responses/1.1-Week 1 Step 1.md:68]: This isn't helpful

**Comment by emsesc** [comment]
@lazyplatypus Please review - made changes requested in the comments

> - add info about prs, forks, branches
> - add instructions to commit code through commandline
> - removed unnecessary section

**Pull request merged**